core/collector.class.inc.php

```diff
@@ -1004,14 +1004,15 @@ protected function CheckColumns($aSynchroColumns, $aColumnsToIgnore, $sSource)
 		}
 	}
 
-	/**
-	 * Check if the collector can be launched
-	 *
-	 * @return bool
-	 */
-	public function CheckToLaunch(): bool
-	{
-		return true;
-	}
+    /*
+     * Check if the collector can be launched
+     *
+     * @param $aOrchestratedCollectors = list of collectors already orchestrated
+     *
+     * @return bool
+     */
+    public function CheckToLaunch($aOrchestratedCollectors): bool {
```

Comment:
```suggestion
    public function CheckToLaunch(array $aOrchestratedCollectors): bool {
```

Also please document the param type and structure in the PHPDoc. See https://blog.jetbrains.com/phpstorm/2022/02/phpstorm-2022-1-eap-3/#multiline_and_nested_array_shapes

Reply:
Has any collector using this new collection plan method been released yet?